docker/docker-compose.yaml

```diff
@@ -0,0 +1,170 @@
+version: '3'
+services:
+    zookeeper:
+        image: wurstmeister/zookeeper
```

Comment:
Huh. I had somehow missed that there was an official zookeeper image. I'll have to have a play and get that working.